packages/forest_admin_datasource_toolkit/lib/forest_admin_datasource_toolkit/components/actions/action_field_factory.rb

```diff
@@ -22,6 +22,9 @@ def self.build_layout_element(field)
             return ActionLayoutElement::RowElement.new(**field) unless field[:fields].empty?
 
             nil
+          when 'Page'
+            # is it nullable if no elements ???
```

Comment:
Yes remove the page if it doesn't contains any element:
https://github.com/ForestAdmin/agent-nodejs/blob/main/packages/_example/src/forest/customizations/card.ts#L75

Reply:
fixed :)